index.html

```diff
@@ -275,5 +275,54 @@ <h1>About</h1>
                 </div>
             </div>
         </section>
+
+        <!-- Contact form -->
+        <section>
+            <!-- Photo at the top -->
+            <div class="contain-img1">
+                <img class="photo-2 " src="/assets/image_geometry_1.png" alt="Background photo">
+            </div>
+
+            <!-- Get stared -->
+            <div class="submit-form">
+                <h1 class="main-name">Get started</h1>
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and using another hx tag (`<h2>`, `<h3>`, etc ...) to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> Note: you are using `h1` tag here ```<h1 class="main-name">Erik Stoupignan</h1>```, that's good because this is the main title 💯👌

Reply:
OK, thank you for your comment.